build.sbt

```diff
@@ -81,7 +81,7 @@ val customMergeStrategy: String => MergeStrategy = {
         MergeStrategy.filterDistinctLines
       case _ => MergeStrategy.deduplicate
     }
-  case "asm-license.txt" | "overview.html" =>
+  case "asm-license.txt" | "overview.html" | "cobertura.properties" =>
```

Comment:
Is there a legit reason the `cobertura.properties` files are included in those jars in the first place?